src/main/java/io/pivotal/cfapp/repository/R2dbcPoliciesRepository.java

```diff
@@ -74,6 +76,15 @@ public R2dbcPoliciesRepository(
                 .then();
     }
 
+    public Mono<Void> deleteMessagePolicyById(String id) {
```

Comment:
I know we don't yet have unit tests for each policy type, but we really should add one.  Happy to collaborate on authoring one with you.  It would go in here: https://github.com/pacphi/cf-butler/tree/master/src/test/java/io/pivotal/cfapp/repository.